index.js

```diff
@@ -2900,6 +2947,14 @@
 
   return S;
 
+  };
+
+  //  Export two versions of the Sanctuary module: one with type checking;
+  //  one without.
+  var S       = createSanctuary(true);
+  S.unchecked = createSanctuary(false);
+  return S;
+
```

Comment:
> Does it not work like that?

For Node, we can do just as you suggest. For the browser, though, we want to provide a single source file so we'd need to use one of the various bundlers. There's nothing to prevent us from doing this, but there's an undeniable appeal to choosing an approach which works the same way for all environments without accommodation.